Editor/Inspector/Menu/MenuItemGUI.cs

```diff
@@ -215,11 +220,46 @@ public void DoGUI()
             EditorGUILayout.BeginHorizontal();
 
             EditorGUILayout.BeginVertical();
-            EditorGUI.BeginChangeCheck();
-            EditorGUILayout.PropertyField(_name, G("menuitem.prop.name"));
-            if (EditorGUI.EndChangeCheck())
+
+            EditorGUILayout.BeginHorizontal();
+            var needsRichLabel = (!string.IsNullOrEmpty(_prop_label.stringValue) || _isTryingRichLabel);
```

Comment:
Would it be simpler to just do `_isTryingRichLabel |= !string.IsNullOrEmpty(_prop_label.stringValue)`?